legacy/README.md

```diff
@@ -12,6 +12,6 @@ unzip -o checkpoints.zip
 
 To back populate collection with past version you can use following bash:
 ```bash
-bash generate_checkpoints.sh 1.0.2 1.0.3 1.0.4
+bash generate_checkpoints.sh "1.0.0" "1.0.1" "1.0.2" "1.0.3" "1.0.4" "1.0.5" "1.0.6" "1.0.7" "1.0.8" "1.1.0" "1.1.1" "1.1.2" "1.1.3" "1.1.4" "1.1.5" "1.1.6" "1.1.7" "1.1.8" "1.2.0" "1.2.1" "1.2.2" "1.2.3" "1.2.4" "1.2.5" "1.2.6" "1.2.7" "1.2.8" "1.2.10" "1.3.0" "1.3.1" "1.3.2" "1.3.3" "1.3.4" "1.3.5" "1.3.6" "1.3.7" "1.3.8"
```

Comment:
We can auto-generate this list with `pip` directly ?

Reply:
this is just temp, for me generating all past, the automated takes only actual release